modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -87,6 +93,12 @@ public void processOpts() {
         if (additionalProperties.containsKey(USE_RX_JAVA)) {
             this.setUseRxJava(Boolean.valueOf(additionalProperties.get(USE_RX_JAVA).toString()));
         }
+        if (additionalProperties.containsKey(USE_RX_JAVA2)) {
+            this.setUseRxJava2(Boolean.valueOf(additionalProperties.get(USE_RX_JAVA2).toString()));
+        }
```

Comment:
If both USE_RX_JAVA and USE_RX_JAVA2 are incorrectly set to true, maybe it would be a good idea to log a warning/error and only set one of them to true only.